src/appimagetool.c

```diff
@@ -457,6 +459,41 @@ bool readFile(char* filename, int* size, char** buffer) {
     return TRUE;
 }
 
+/* run a command outside the current appimage, block environs like LD_LIBRARY_PATH */
+int run_external(const char *filename, char *const argv []) {
+    int pid = fork();
+    if (pid == -1) {
```

Comment:
I think you could/should check `<= 0` here.

Reply:
`< 0` I suppose?